### ISSUE TITLE
COMException: The RPC server is unavailable. (Exception from HRESULT: 0x800706BA)

### ISSUE THREAD

**Issue by nint8835** [nint8835/iTunesRichPresence#21]
https://sentry.io/riley-flynn/itunesrichpresence/issues/577328048/

```
COMException: The RPC server is unavailable. (Exception from HRESULT: 0x800706BA)
  Module "iTunesLib.IiTunes", in get_CurrentTrack
    iTunesLib.IITTrack get_CurrentTrack()
  File "D:\Development\iTunesRichPresence-Rewrite\iTunesRichPresence-Rewrite\DiscordBridge.cs", line 86, in Timer_OnTick
    Void Timer_OnTick(System.Object, System.EventArgs)
  Module "MS.Internal.CulturePreservingExecutionContext", line 27, in CallbackWrapper
    Void CallbackWrapper(System.Object)
  Module "MS.Internal.CulturePreservingExecutionContext", line 0, in Run
    Void Run(MS.Internal.CulturePreservingExecutionContext, System.Threading.ContextCallback, System.Object)
  Module "MS.Win32.HwndWrapper", line 48, in WndProc
    IntPtr WndProc(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
...
(24 additional frame(s) were not displayed)

The RPC server is unavailable. (Exception from HRESULT: 0x800706BA)
```

**Comment by nint8835** [comment]
Occurs when iTunes is closed and app attempts to retrieve information from it.

**Comment by nint8835** [comment]
Fixed with commit afb38c1
